ocaml/sdk-gen/go/gen_go_binding.ml

```diff
@@ -55,7 +55,15 @@ let main destdir =
         render_template "FileHeader.mustache" obj ~newline:true ()
       in
       let record_rendered = render_template "Record.mustache" obj () in
-      let rendered = header_rendered ^ record_rendered in
+      let methods_rendered = render_template "Methods.mustache" obj () in
+      let rendered =
+        let first_half = header_rendered ^ record_rendered in
+        match methods_rendered with
```

Comment:
I understand if an `obj` contains any messages can be determined by the structure of the `obj`. It may not be necessary to wait for a result of rendering.

Reply:
I have added `~strict:true`.